bin/prettier.js

```diff
@@ -351,12 +358,20 @@ if (stdin) {
         process.exitCode = 2;
       }
     } else if (!argv["list-different"]) {
-      // Don't use `console.log` here since it adds an extra newline at the end.
-      process.stdout.write(output);
+      writeOutput(result);
     }
   });
 }
 
+function writeOutput(result) {
+  if (options.cursorOffset) {
+    console.log(result.cursorOffset);
```

Comment:
Oh, good idea, I hadn't thought of that. Fixed in https://github.com/prettier/prettier/pull/1637/commits/8f72d51a901870697005ebb6ca06eb86f398f6aa